app/src/main/java/com/babylon/wallet/android/presentation/transfer/accounts/ChooseAccountSheet.kt

```diff
@@ -340,7 +340,8 @@ fun ScanQRContent(
                 .fillMaxWidth()
                 .aspectRatio(1f)
                 .padding(horizontal = RadixTheme.dimensions.paddingDefault)
-                .clip(RadixTheme.shapes.roundedRectMedium)
+                .clip(RadixTheme.shapes.roundedRectMedium),
+            disableBackHandler = false
```

Comment:
Why is this needed?

Reply:
otherwise you can't use back button on camera preview within the sheet